packages/twentytwenty-theme/src/components/archive/archive.js

```diff
@@ -52,8 +65,15 @@ const Archive = ({ state, showExcerpt, showMedia }) => {
           </Fragment>
         );
       })}
-      <Pagination />
-    </>
+      {state.theme.infiniteScrolling ? (
```

Comment:
@luisherranz I think I'm using the state / settings correctly here? I tested it and it seems okay

Reply:
Yes, sure. That's the way settings work :)